Description: Don't sign assemblies, as they're not GAC'd. To be removed if this
             ever gets a tarball release that GAC's the assemblies
From: Chow Loong Jin <hyperair@ubuntu.com>
Forwarded: not-needed
Index: clutter-sharp/clutter-gtk/AssemblyInfo.cs.in
===================================================================
--- clutter-sharp.orig/clutter-gtk/AssemblyInfo.cs.in	2011-03-20 01:44:49.011292799 +0800
+++ clutter-sharp/clutter-gtk/AssemblyInfo.cs.in	2011-03-20 01:44:56.184586318 +0800
@@ -2,5 +2,3 @@
 using System.Runtime.CompilerServices;
 
 [assembly:AssemblyVersion ("@ASSEMBLY_VERSION@")]
-[assembly:AssemblyDelaySign (false)]
-[assembly:AssemblyKeyFile ("@top_srcdir@/clutter-sharp.snk")]
Index: clutter-sharp/clutter/AssemblyInfo.cs.in
===================================================================
--- clutter-sharp.orig/clutter/AssemblyInfo.cs.in	2011-03-20 01:47:02.210553496 +0800
+++ clutter-sharp/clutter/AssemblyInfo.cs.in	2011-03-20 01:47:08.857183272 +0800
@@ -2,5 +2,3 @@
 using System.Runtime.CompilerServices;
 
 [assembly:AssemblyVersion ("@ASSEMBLY_VERSION@")]
-[assembly:AssemblyDelaySign (false)]
-[assembly:AssemblyKeyFile ("@top_srcdir@/clutter-sharp.snk")]
